client/src/components/widgets/Plots/script.js

```diff
@@ -194,21 +194,22 @@ export default {
           }
           return step;
         });
-      if (!this.isTimeStepLoaded(firstAvailableStep)) {
-        await this.fetchTimeStepData(firstAvailableStep);
+      await this.fetchTimeStepData(firstAvailableStep);
```

Comment:
No, we only need to remove this check because we need to correctly set the plot type for a new plot. At 303 the plot type has already been established and we can avoid attempting to re-fetch data we already have with the `isTimeStepLoaded` check. The `PlotFetcher` class does not currently take into account if there are multiple instances of the same plot so we still need this check.

Reply:
:+1: